.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Writer's Quality Checklist
+
+Before merging your PR, did you:
+
+- [ ] Run spell checker
+- [ ] Run link checker to check for broken xrefs
+- [ ] Check for orphan files
+- [ ] Perform a local build and do a final visual check of your content, including checking for:
+  - Broken images
+  - Dead links
+  - Correct rendering of partials if they are used in your content
+  - Formatting issues, such as:
+    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
+    - Messed up tables
+    - Proper indentation
+    - Correct header levels
+- [ ] Receive final review and signoff from:
+  - Technical SME
+  - Product Manager
+  - Editor or peer reviewer
+  - Reporter, if this content is in response to a reported issue (internal or external feedback)
+- [ ] Verify that the software actually got released
```

Comment:
Re "- [ ] Verify that the software actually got released" 
this one might not be applicable in some/many situations. Often we're fixing bugs or adding examples or more detail about a product that has already been release.

Reply:
Thanks. I just merged your suggestion at 6f66f64 which should address this concern.